component/javascript/index.js

```diff
@@ -0,0 +1,17 @@
+const snackbarShowBtn = document.querySelector(".snackbar-show-btn");
```

Comment:
if there is only one file, you can put the js directly under the folder, and its common to name folder as **js** even if you want, dint see anybody naming **javascript**